<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>181</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="button_angulo">
       <property name="text">
        <string>set ángulo [°]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_coeficiente">
       <property name="text">
        <string>set Coeficiente μ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_masa">
       <property name="text">
        <string>set Masa [kg]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_base_obj_2">
       <property name="text">
        <string>set Base objeto [m]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_altura_obj">
       <property name="text">
        <string>set Altura objeto [m]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_altura_force">
       <property name="text">
        <string>set Altura Fuerza [m]</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>137</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QDoubleSpinBox" name="spin_box_ang">
       <property name="minimum">
        <double>-9999999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="dspinbox_coef">
       <property name="minimum">
        <double>-9999999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="dspinbox_m">
       <property name="minimum">
        <double>-99999999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="dspinbox_bo">
       <property name="minimum">
        <double>-9999999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="dspinbox_ao">
       <property name="minimum">
        <double>-9999999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="dspinbox_af">
       <property name="minimum">
        <double>-9999999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999999999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="container_abc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>429</y>
      <width>1211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Mensaje de alerta</string>
    </property>
    <widget class="QLabel" name="Alertmsg_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>1191</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;dsds&lt;/p&gt;&lt;p&gt;sdsadsa&lt;/p&gt;&lt;p&gt;sdsadsa&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonsimulate">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SIMULAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>set Direcció de la fuerza</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="set_direccion">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>15</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;VARIABLES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>61</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="ok_1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ok_2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ok_3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ok_4">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ok_5">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ok_6">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>731</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QLabel" name="okissue_7">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>57</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1234</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
